.github/workflows/pull_requests.yaml

```diff
@@ -66,7 +66,6 @@ jobs:
       - name: compare console values with main
         run: ./dyff --color=off -k between -s <(git show 'origin/main:charts/console/values.yaml') charts/console/values.yaml
   test:
-    needs: lint
```

Comment:
why are we removing this line? We should avoid proceeding if linter fails.

Reply:
Because they run in parallel. If the linter fails, everything else will get killed anyway. Why make us wait?

Reply:
Linter and check-values run in parallel, not the matrix tests? from past CI runs we had linter failed but the other tests continue. Did you change some setting to fail if any tests fail? Maybe i missed this.

Reply:
Damn, you're right. Jobs can't cancel each other. There are tools to do that, but I'll just put it back for now.